templates/nutrient_levels.tt.html

```diff
@@ -0,0 +1,28 @@
+<!-- Nutrient levels template -->
+[% IF (nutrition_grade == "exists") && (nutrient_levels != '') %]
```

Comment:
With this if statement, you have to repeat the template twice. I think we should remove the if statement, especially as it is still in the Perl code.

What it means is that this template should display only the nutrient levels, but not the "<div class="small-12 xlarge-6 columns">" that may be added (but not always) before it.